### PR TITLE
Move primitives material and Optimize to Run

### LINKED ISSUE #39: [ MOVE ] "Build with Primitives" section from Build to Run
The following pages currently in the Build section should be moved to the Run section:
- Get Started with Primitives
- Primitives Examples
- Specify advanced runtime options

As part of this the introduction page of build should be updated to no lnger talk about primitives as part of the build section

### LINKED ISSUE #110: [ MOVE ] Optimize section into Run section
The Optimise section should be removed and the pages transferred to the Run section. These pages should be moved into a new subsection within Run, called "optimise" or something better

### PR DESCRIPTION
**Pull request by abbycross**
Closes #39 
Closes #110

### CODE REVIEW

**Comment by abbycross** [comment]
@Eric-Arellano I will need help figuring out how to remove the Optimize tab in the UI. I am not sure what file(s) it lives in. This also affects the [Overview page](https://docs.quantum-computing.ibm.com/), where there is an Optimize card that will have to be removed.

**Comment by abbycross** [comment]
To streamline the toc updates without causing a multitude of merge conflicts, I am moving items both from Optimize and from Build into the run section in this PR. I can't remove the Optimize section altogether until we have help removing its tab from the UI and its card from the Overview page.

File: docs/build/index.mdx [hunk 1]
```diff
@@ -6,7 +6,4 @@ description: Build circuits with Qiskit and Qiskit Runtime
 
 # Build circuits with Qiskit
 
-In the build phase, you create quantum programs that represent the problem you are solving. The foundation of quantum programs are quantum circuits, which are made up of gates (or operations) that are used to manipulate qubits in the quantum computer. You can also use _primitives_, which provide methods that make it easier to build modular algorithms and other higher-order programs, as well as providing the interface to run those circuits on a backend and get outputs in a specific format.  Depending on your needs, you can use Qiskit or OpenQASM to create your circuits, followed by Qiskit Runtime to initialize your chosen primitive for an IBM Quantum backend. In summary, the build phase consists of these processes:
-
-1. Defining quantum circuits
-2. Initializing primitives
+In the build phase, you create quantum programs that represent the problem you are solving. The foundation of quantum programs are quantum circuits, which are made up of gates (or operations) that are used to manipulate qubits in the quantum computer.  Depending on your needs, you can use Qiskit or OpenQASM to create your circuits.
```
Comment (javabster) [inline docs/build/index.mdx:9]: This intro is good for me but @blakejohnson do you have thoughts?

Comment (javabster) [reply docs/build/index.mdx:9]: It does feel a little short now that we've removed the primitives stuff 🤔

**Comment by abbycross** [comment]
I need to integrate the toc changes from work @beckykd is doing in parallel, so converting this back to a draft

File: docs/run/_toc.json [hunk 2]
```diff
@@ -43,28 +53,63 @@
...
+        {
+          "title": "Retired systems",
+          "url": "/run/retired-systems"
+        }
+      ]
     },
     {
-      "title": "Retired systems",
-      "url": "/run/retired-systems"
+      "title": "Understand the plaform",
```
Comment (abbycross) [inline docs/run/_toc.json:94]:
```suggestion
      "title": "Understand the platform",
```

File: docs/build/index.mdx (hunk 1 above)
Comment (abbycross) [reply docs/build/index.mdx:9]: Along the same lines, we may want to add primitives info to the Run introduction page -- or should it stay the same? Here's how it looks now
Screenshot 2023-10-10 at 12 16 31 PM

**Comment by abbycross** [comment]
I propose we move the discussions about the Start and Build intro pages to a new PR, so that this one can be merged. Other PRs are waiting on it since the TOCs are undergoing such a big change. @javabster @beckykd @Eric-Arellano can someone please approve?

**Comment by abbycross** [comment]
Issue to rewrite intro pages is here: https://github.com/Qiskit/documentation/issues/143

**Review by Eric-Arellano — APPROVED** [review]
Approving since we can make follow up improvements in https://github.com/Qiskit/documentation/issues/143

File: docs/build/index.mdx (hunk 1 above)
Comment (blakejohnson) [reply docs/build/index.mdx:9]: Gates are not the only valid operations in a quantum circuits. So, I would suggest something to the effect of:
> are made up of operations including gates, measurement, and reset of qubits.

Comment (blakejohnson) [reply docs/build/index.mdx:9]: Ok, so some other things I think we need to do in the intro are to talk about what it is we are building. This effectively should signpost what is to come in the section. i.e. something like:

All tasks require building one or more quantum circuits. Some tasks additionally require constructing quantum operators to define properties of quantum states that we want to estimate/measure.

Qiskit enables working with circuits (and, to a lesser extent, operators) at various abstraction levels:
* abstract
* virtual
* physical
* scheduled
* pulse program
At the most **abstract** level, we have a task-oriented lens in the circuit library. We can also express operations in abstract mathematical terms using Operators, isometries, and classical/boolean functions. Then we have **virtual** circuits where those mathematical abstractions take on a concrete representation in terms of a concrete gate set. At the **physical** level those instructions are mapped to specific physical qubits and instructions are re-written to reflect the connectivity and native gate set of a target hardware platform. **Scheduled** circuits introduce timing information. And finally **pulse programs** represent signals on channels. 

Qiskit, and OpenQASM, further support the notion of _extended_ circuits which expand the set of allowed operations to include real-time computations on classical values. Qiskit's tooling for working with this richer family of circuits is found in the section on Dynamic Circuits.

Comment (abbycross) [reply docs/build/index.mdx:9]: Thanks Blake -- moving your comments into #143 now that we have an intro-page-specific PR.

**Pull request merged**